web/portal/brand/src/entities/Administrator/Action/Impersonate.tsx

```diff
@@ -25,16 +27,26 @@ const Impersonate: ActionFunctionComponent = (props: ActionItemProps) => {
   const { username, active } = row;
   const isActionDisabled = customData === undefined || !active;
 
-  if (isActionDisabled) {
+  if (isActionDisabled || !canImpersonate) {
     return (
       <>
         {variant === 'text' && (
           <MoreMenuItem disabled={true}>{_('Impersonate')}</MoreMenuItem>
         )}
         {variant === 'icon' && (
-          <StyledTableRowCustomCta disabled={true}>
-            <AdminPanelSettingsIcon />
-          </StyledTableRowCustomCta>
+          <>
+            <Tooltip
+              title={_('Impersonate')}
+              placement='bottom-start'
+              enterTouchDelay={0}
+            >
+              <span>
+                <StyledTableRowCustomCta disabled={true}>
+                  <AdminPanelSettingsIcon />
+                </StyledTableRowCustomCta>
+              </span>
+            </Tooltip>
+          </>
```

Comment:
```suggestion
          <Tooltip
            title={_('Impersonate')}
            placement='bottom-start'
            enterTouchDelay={0}
          >
            <span>
              <StyledTableRowCustomCta disabled={true}>
                <AdminPanelSettingsIcon />
              </StyledTableRowCustomCta>
            </span>
          </Tooltip>
```

...or you will end with <> twice.